ansible.html.markdown

```diff
@@ -442,7 +442,7 @@ In fact - jinja is used to template parts of the playbooks too
 ```
 
 #### Jinja2 filters
-Junja is powerfull. It has built-in many usefull functions.
+Jinja is powerful. It has built-in many useful functions.
```

Comment:
It has many built-in useful functions? Or even better: It has many useful built-in functions.